src/js/components/UnitsHealthNodeDetail.js

```diff
@@ -84,6 +85,11 @@ class UnitsHealthNodeDetail extends mixin(StoreMixin) {
     const unitSummary = UnitSummaries[unit.get("id")] || {};
     const unitDocsURL =
       unitSummary.getDocumentationURI && unitSummary.getDocumentationURI();
+    const joinedUnitSummary = unitSummary.summary
```

Comment:
`TypeError: unitSummary.summary is undefined, can't access property "map" of it` to repro please check `DC/OS Checks Timer (Healthy)`

Reply:
The same thing happens in https://github.com/dcos/dcos-ui/pull/3367 .

Reply:
#3367 is based on this branch to get the changes to `ErrorMessageUtil` variable format. I will rebase changes from this branch as they are made.